src/backend/web/graphql/index.js

```diff
@@ -36,9 +36,38 @@ module.exports.typeDefs = graphql`
     getFeeds: [Feed]
     getFeedsCount: Int
     getPost(id: ID!): Post
-    getPosts(page: Int, perPage: Int): [Post]
+    getPosts(filter: PostFilter, page: Int, perPage: Int): [Post]
     getPostsCount: Int
   }
+
+  # Post filters
+  input PostFilter {
+    author: String
+    fromDate: Int
+    toDate: Int
```

Comment:
Why use `Int` for these dates?  Won't most callers of this expect to be able to use Date objects?